README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://readthedocs.org/projects/serenata-toolbox/badge/?version=latest
+   :target: http://serenata-toolbox.readthedocs.io/en/latest/?badge=latest
+   :alt: Documentation Status
+
+.. image:: https://landscape.io/github/jonDel/serenata-toolbox/master/landscape.svg?style=flat
+   :target: https://landscape.io/github/jonDel/serenata-toolbox/master
```

Comment:
Can you change `jonDel` to `datasciencebr`? I believe I just authorized the service.